pep-0517.txt

```diff
@@ -744,12 +744,13 @@ across the ecosystem.
 more powerful options for evolving this specification in the future.
 
 For concreteness, imagine that next year we add a new
-``prepare_wheel_metadata2`` hook, which replaces the current
-``prepare_wheel_metadata`` hook with something that produces more data, or a
-different metadata format. In order to
-manage the transition, we want it to be possible for build frontends
-to transparently use ``prepare_wheel_metadata2`` when available and fall
-back onto ``prepare_wheel_metadata`` otherwise; and we want it to be
+``build_sdist_from_vcs`` hook, which provides an alternative to the current
+``build_sdist`` hook where the frontend is responsible for passing
+version control tracking metadata to backends (including indicating when all
```

Comment:
+1 to cutting this comparison section down to a brief paragraph, or moving it to an appendix or something (can PEPs have appendices?). It's a large amount of text which doesn't really help to describe what the PEP is.